cypress/e2e/models/administration/credentials/credentialsJira.ts

```diff
@@ -0,0 +1,108 @@
+import { Credentials } from "./credentials";
+import { CredentialsJiraData } from "../../../types/types";
+import {
+    cancelForm,
+    click,
+    clickByText,
+    exists,
+    inputText,
+    notExists,
+    submitForm,
+    validateValue,
+} from "../../../../utils/utils";
+import { passwordInput, usernameInput } from "../../../views/credentials.view";
+import { button, CredentialType } from "../../../types/constants";
+
+export class CredentialsJira extends Credentials {
+    type = CredentialType.jira;
+    email: string;
+    token: string;
+
+    constructor(credentialsJiraData: CredentialsJiraData) {
+        super();
+        this.init(credentialsJiraData);
+    }
+
+    protected init(credentialsJiraData: CredentialsJiraData) {
+        const { name, description, email, token } = credentialsJiraData;
+        this.name = name;
+        this.description = description;
+        this.email = email;
+        this.token = token;
+    }
+
+    protected fillEmail() {
+        inputText(usernameInput, this.email);
+    }
+
+    protected fillToken() {
+        inputText(passwordInput, this.token);
+    }
+
+    protected selectCredType() {
+        click("#user-credentials-select-toggle");
+        clickByText(button, "Username/Password");
+    }
+
+    create(toBeCanceled = false) {
+        super.create();
+        super.create();
```

Comment:
**small issue:** Here are two sequential calls to the same method

Reply:
Thanks, missed that mistake. Fixed already :-)